src/main/java/nextstep/subway/line/domain/Sections.java

```diff
@@ -20,46 +21,55 @@ public Sections() {
     }
 
     public List<Section> getSections() {
-        return sections;
+        return sections.stream()
+                .sorted((s1, s2) -> {
+                    if (s1.getDownStation().equals(s2.getUpStation())) {
+                        return -1;
+                    }
+                    return 0;
+                })
+                .collect(Collectors.toList());
     }
 
     public List<Station> getStations() {
-        List<Station> stations = sections.stream().map(Section::getUpStation).collect(Collectors.toList());
+        List<Station> stations = getSections().stream().map(Section::getUpStation).collect(Collectors.toList());
```

Comment:
개행으로 가독성을 높이면 어떨까요?
IntelliJ를 사용하신다면 각 메서드마다의 리턴 타입도 확인할 수 있어요
```suggestion
        List<Station> stations = getSections().stream()
            .map(Section::getUpStation)
            .collect(Collectors.toList());
```